blivet/devicefactory.py

```diff
@@ -1752,6 +1759,92 @@ def _get_new_device(self, *args, **kwargs):
         return super(LVMThinPFactory, self)._get_new_device(*args, **kwargs)
 
 
+class LVMVDOFactory(LVMFactory):
+
+    """ Factory for creating LVM VDO volumes.
+    """
+
+    def __init__(self, storage, **kwargs):
+        self.pool_name = kwargs.pop("pool_name", None)
+        self.virtual_size = kwargs.pop("virtual_size", None)
```

Comment:
Might be nice to (re)document here what happens if `pool_name` and/or `virtual_size` are omitted.

Reply:
I added the LVM VDO specific keyword arguments to the docstring.